huaweicloud/resource_huaweicloud_identity_agency.go

```diff
@@ -437,10 +459,15 @@ func resourceIAMAgencyV3Update(d *schema.ResourceData, meta interface{}) error {
 
 	agencyID := d.Id()
 
-	if d.HasChanges("delegated_domain_name", "description") {
+	if d.HasChanges("delegated_domain_name", "description", "duration") {
```

Comment:
should put "delegated_service_name" in the list

Reply:
thanks, 'delegated_service_name' is putting into the list